### DISCUSSION TITLE
This branch cannot be rebased safely

### DISCUSSION THREAD

**Discussion by carlescufi** [community/community#22403] [New to GitHub]
<p>Hi all,</p>
<p>Has anyone noticed the new error message on a rebase-based workflow:</p>
<p><strong>This branch cannot be rebased safely</strong></p>
<p>Rebasing the commits of this branch on top of the base branch cannot be performed automatically as this would create a different result than a regular merge.</p>
<p>Apparently this blocks PRs from being rebased on top of the target branch instead of merged, and has broken our rebase workflow.</p>
<p>Any ideas on what is the purpose of this?</p>
<p>Thanks!</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/carlescufi">@carlescufi</a>,</p>
<p>It’s hard to tell why you’re seeing this without knowing which repository you’re seeing it in. Even then, we might need you to reach out to private support in order to investigate appropriately (we may need to ask for private information we wouldn’t want to discuss in a public forum). To be safe, I’d recommend reaching out to our private support team at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>.</p>
<p>Cheers!</p>

**Comment by carlescufi** [comment, marked as answer]
<p>For the future: This happens if you have a merge commit in a Pull Request inside a repo that only allows rebase merging as the strategy in the merge button.</p>
